### DISCUSSION TITLE
New Add-on idea

### DISCUSSION THREAD

**Discussion by startFX** [ScratchAddons/ScratchAddons#4063] [Ideas]
You could add an Add-on that allows you to change the delay between autosaves, this would be very useful.
If the Scratch Addons team reads this, thank you for making Scratch more awesome ;)

**Comment by TheColaber** [comment]
Maybe turn disable auto save into "Change auto-save" or something like that, where you can change the time it takes for auto-save to happen or just disable it altogether.

**Reply by startFX** [reply]
ok
